base/sysimg.jl

```diff
@@ -587,6 +587,7 @@ end
     @deprecate_binding Terminals       root_module(Base, :REPL).Terminals       true ", use `REPL.Terminals` instead"
 
     @deprecate_binding Pkg root_module(Base, :Pkg) true ", run `using Pkg` instead"
+    @deprecate_binding OldPkg root_module(Base, :OldPkg) true ", run `using OldPkg` instead"
```

Comment:
This seems unlikely to happen, no? We can probably just skip this I would think.